### PR TITLE
Enhancement: Add Sphinx build

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: missing-env-variable-logs.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ Building application 'myapp' (runtime type: php:7.3, tree: abcd123)
     W:
     W:   In EnvVarProcessor.php line 76:
     W:
-    W:     Environment variable not found: "DATABASE_URL".
+    W:     Environment variable not found: "DATABASE_URL".git
```
Comment (fabpot) [inline missing-env-variable-logs.txt:31]: Are you sure this is needed?

Comment (OskarStark) [reply missing-env-variable-logs.txt:31]: Oh, no

Comment (OskarStark) [reply missing-env-variable-logs.txt]: Fixed

**Comment by fabpot** [comment]
Thank you so much Oskar!

**Comment by OskarStark** [comment]
You are welcome 🙏

**Pull request merged**